vars/Debian.yml

```diff
@@ -0,0 +1,6 @@
+---
+pulp_preq_packages:
+  - python3
+  - python3-venv
+
+pulp_psycopg_package: 'python{% if ansible_python.version.major != 2 %}{{ ansible_python.version.major }}{% endif %}-psycopg2'
```

Comment:
Pulp 3 requires Python 3. Shouldn't python3-psycopg2 always be installed?

Reply:
see above :)